deploy/template.yaml

```diff
@@ -107,6 +107,18 @@ Parameters:
     Description: >
       The ARN of the Code Signing Config to use, provided by the deployment pipeline
     Default: "none"
+  DeploymentStrategy:
```

Comment:
is this not used elsewhere in the template?

Reply:
It should be, however the place where it will be used is part of the ticket OLH-1893 which is not yet merged. So blocking the ticket for this until that is done.